ajax/libs/instagram-lite/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "instagram-lite",
+  "version": "2.0.0",
+  "filename": "instagramLite.min.js",
+  "description": "A simple, lightweight jQuery plugin used to display a user's Instagram photos.",
+  "keywords": [
+    "instagram",
+    "photos",
+    "api",
+    "feed"
+  ],
+  "license": "MIT",
+  "author": {
+    "name": "Michael Lynch",
+    "url": "http://michaelynch.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/michael-lynch/instagram-lite"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/michael-lynch/instagram-lite.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "src/instagramLite.js",
+          "src/css/style.css",
+          "build/instagramLite.min.js"
+        ]
```

Comment:
Hosting the source files doesn't make sense. All we need to host is `instagramLite.min.js` located in the `build` directory. Change this to:

```
"files": [
  "**/*"
]
```

Since the `build` directory will always contain the final, CDN-compatible file we can include all its contents.